neqo-transport/src/send_stream.rs

```diff
@@ -552,6 +561,8 @@ pub struct SendStream {
     priority: TransmissionPriority,
     retransmission_priority: RetransmissionPriority,
     retransmission_offset: u64,
+    bytes_written: u64,
+    bytes_sent: u64,
```

Comment:
Do we need to track this here?

Edit: see comments below...

bytes_written => retired + buf.len(), then you need to track a value
bytes_sent => fc.used(), then final_size